resource_alks_iamrole.go

```diff
@@ -199,6 +206,17 @@ func resourceAlksIamRoleRead(ctx context.Context, d *schema.ResourceData, meta i
 	_ = d.Set("arn", foundRole.RoleArn)
 	_ = d.Set("ip_arn", foundRole.RoleIPArn)
 	_ = d.Set("enable_alks_access", foundRole.AlksAccess)
+	_ = d.Set("role_added_to_ip", foundRole.RoleAddedToIP)
+
+	var roleType = d.Get("type")
+	if (roleType == nil) || (roleType == "") {
+		jsonStrPolicy, err := json.Marshal(foundRole.TrustPolicy)
+		if err == nil {
+			_ = d.Set("assume_role_policy", string(jsonStrPolicy))
+		}
+		_ = d.Set("include_default_policies", false)
+	}
+	_ = d.Set("max_session_duration_in_seconds", foundRole.MaxSessionDurationInSeconds)
```

Comment:
I pondered over this for a while. I had to look at the test to remind myself of the alksGo functionality. Looks good